UnitTesting.py

```diff
@@ -0,0 +1,22 @@
+import unittest
+from CheckPart import isFatherAlive
+from spouseOrParentShares import calculate_and_deduct, SpouseWithSonsOrDaughters
+
+class TestFamilyStatus(unittest.TestCase):
```

Comment:
Please revisit all Classes, variables, methods, functions, and file names to make it look like python. Some file names are in small alphabets, some are in capital, some have `_` some are in CamelCase. This is too inconsistent.

Reply:
I visit my files,now they are formatted accordingly..